roles/installer/defaults/main.yml

```diff
@@ -6,6 +6,9 @@ api_version: '{{ deployment_type }}.ansible.com/v1beta1'
 database_name: "{{ deployment_type }}"
 database_username: "{{ deployment_type }}"
 
+# Enable nginx IPv6 binding on default
+ipv6_disabled: no
```

Comment:
This would need to be configurable via the AWX spec.  In it's current state, I don't see how this PR would allow users to toggle this setting (for example if they have an environment that only allows ipv6).

Reply:
You'll also need to rebase to resolve the conflicts on this PR before merging.  

@digifuchsi @MaxVolum3 @koshrf I am sorry this went so long without a review.  We now have weekly triage meetings for community PR's, so contributions like this will not sit for so long in the future.

Reply:
@digifuchsi aside from that, this PR looks good.  With those changes we should be able to get this merged.  Please drop me a ping here when you've added those changes.